packages/styles/src/components/accordion.scss

```diff
@@ -1,94 +1,124 @@
 @forward './../variables/options';
 
-@use './../lic/bootstrap-license';
-@use './../themes/bootstrap/core' as *;
-@use './../themes/bootstrap/accordion' as ba;
+@use './../mixins/button' as button-mixins;
+@use './../mixins/color' as color-mixins;
+@use './../mixins/icons' as icon-mixins;
+@use './../mixins/type' as type-mixins;
+@use './../mixins/utilities' as utility-mixins;
 
-@use './../mixins/type';
-@use './../mixins/utilities';
-@use './../variables/spacing';
 @use './../variables/components/accordion';
-@use './../variables/components/button';
 
 .accordion-item {
-  border-right: 0;
-  border-left: 0;
-  background-color: transparent;
+  --post-accordion-border-width: #{accordion.$accordion-border-width};
+  --post-accordion-border-color: #{accordion.$accordion-border-color};
+  --post-accordion-header-font-weight: #{accordion.$accordion-header-font-weight};
+  --post-accordion-header-line-height: #{accordion.$accordion-header-line-height};
+  --post-accordion-button-padding: #{accordion.$accordion-button-padding};
+  --post-accordion-icon-size: #{accordion.$accordion-icon-size};
+  --post-accordion-button-cursor: #{accordion.$accordion-button-cursor};
+  --post-accordion-button-disabled-opacity: #{accordion.$accordion-button-disabled-opacity};
+  --post-accordion-button-focus-box-shadow: #{accordion.$accordion-button-focus-box-shadow};
+  --post-accordion-icon-transition: #{accordion.$accordion-icon-transition};
+  --post-accordion-icon-transform: #{accordion.$accordion-icon-transform};
+  --post-accordion-body-padding: #{accordion.$accordion-body-padding};
+  --post-accordion-heading-margin: #{accordion.$accordion-heading-margin};
+  --post-accordion-heading-font-weight: #{accordion.$accordion-heading-font-weight};
```

Comment:
Did you define CSS variables because you want to give the users easy access to all of those styles properties?
- If the answer is "no": Why don't you use the SCSS variables directly instead of defining some CSS variables out of it?
- If the answer is "yes": Why don't you define the variables on the host?

Reply:
The answer is "yes", I appreciate the flexibility offered by CSS variables (but I'm open to discussing it further!).

If by "host" you mean the `.accordion`, I didn't declare the variables there because there is no `.accordion` in a collapsible: the `.accordion` is just a wrapper for multiple `.accordion-item`, i.e. several collapsible.

Reply:
Discussion moved to #1380